test/standalone/test-grpc-context.js

```diff
@@ -18,61 +18,158 @@
 var agent = require('../..').start({ samplingRate: 0 }).private_();
 
 var common = require('../hooks/common.js');
-
 var assert = require('assert');
 var express = require('../hooks/fixtures/express4');
 var http = require('http');
-var grcPort = 50051;
-var debugCount = 0;
-agent.logger.debug = function(error) {
-  if (error.indexOf('http') !== -1) {
-    debugCount++;
-  }
-};
 
 var versions = {
   grpc013: require('../hooks/fixtures/grpc0.13'),
   grpc014: require('../hooks/fixtures/grpc0.14'),
   grpc015: require('../hooks/fixtures/grpc0.15')
 };
 
+var grpcPort = 50051;
+var protoFile = __dirname + '/../fixtures/test-grpc.proto';
+var debugCount = 0;
+var client, grpcServer, server;
+
+agent.logger.debug = function(error) {
+  if (error.indexOf('http') !== -1) {
```

Comment:
Could we strengthen this check to ensure the we are only failing to log http requests made to the express app. (Ensure that no requests are associated with the url www.google.com).